src/components/LoginPage/GoogleLogout/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { useGoogleLogout } from "react-google-login";
+
+const clientId =
+  "743068888490-2chsv38s7paclkbc1qjd2ktd3jjf2aqf.apps.googleusercontent.com";
+
+function Logout() {
+  const onLogoutSuccess = (res) => {
+    console.log("Logged out Success");
+    alert("Logged out Successfully");
+  };
+
+  const onFailure = () => {
+    console.log("Handle failure cases");
+  };
+
+  const { signOut } = useGoogleLogout({
+    clientId,
+    onLogoutSuccess,
+    onFailure,
+  });
+
+  return (
+    <button onClick={signOut} className="button">
```

Comment:
Give more meaningful className, button might e used by a library or a different team mate and it’ll make conflicts

Reply:
PTAL